.editorconfig

```diff
@@ -0,0 +1,39 @@
+# https://editorconfig.org/
```

Comment:
I think you shouldn't commit this one, it's very specific for your editor, as it's an open library, they don't need to have those configs

Reply:
I thought that would be a good idea to unify code styles. Because, some files were not under PEP8

Reply:
Yes, I know, but I don't think people will follow it just because we have that file here. That is my point, maybe we should try to force to use it, somehow.

Reply:
I see your point.
I have thoughts about adding pep8 check into travis. What do you think about it?

Reply:
Yeah, it would be nice, or at least a pre-commit check.

Reply:
Done!